### PR TITLE
Date type methods

### PR DESCRIPTION
**Pull request by dmendez**

### CODE REVIEW

File: src/date/fromString.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/**
+ * Returns a Date from string value format espected format dd[/]mm[/]yyyy
+ * @param {string} dateFrom
+ * @return Date
+ */
+
+import { DateTime } from "luxon";
+import { nullValue } from "../generator/out/not_implemented";
```
Comment (mcrispino) [inline src/date/fromString.ts]: `import nullValue` is unnecessary, it is not used.

File: src/date/fromString.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import { EMPTY_DATE_VALUE } from "./core";
+
+export const fromString = (dateFrom: string): Date => {
+  const dateParts = dateFrom.match(
```
Comment (mcrispino) [inline src/date/fromString.ts]: Implementation assumes date format is dd/mm/yyyy. Is it always the case? Couldn't it be using another date format?

File: src/date/toString.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+/**
+ * Returns a string represantation of a date. dd[/]mm[/]yyyy
+ * @return Date
+ */
+import {isEmpty} from './isEmpty';
+
+export const toString = (targetDate: Date): string => {
+  return isEmpty(targetDate) ? "" : `${targetDate.getDate()}/${targetDate.getMonth() + 1}/${targetDate.getFullYear()}`;
```
Comment (mcrispino) [inline src/date/toString.ts]: Again, implementation assumes date format is dd/mm/yyyy

File: src/date/fromString.ts (hunk 1 above)
Comment (dmendez) [reply src/date/fromString.ts]: Fixed at commit d440aae2a58e17de941b6a9953099207128944ec

Comment (dmendez) [reply src/date/fromString.ts]: We need to solve the string representation format for this date values in a model preferences dependant way. 
Issue #33 was created.

File: src/date/fromString.ts (hunk 2 above)
Comment (dmendez) [reply src/date/fromString.ts]: We need to solve the string representation format for this date values in a model preferences dependant way. 
Issue #33 was created.

**Pull request merged**